sdk/core/azure-core-amqp-experimental/pom.xml

```diff
@@ -45,6 +45,9 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <legal><![CDATA[[INFO] Any downloads listed may be third party software. Microsoft grants you no rights for third party software.]]></legal>
+    <skipNewCodesnippetTooling>false</skipNewCodesnippetTooling>
```

Comment:
Is there a reason we explicitly add these parameters and not in the parent?

Reply:
It's to enable porting libraries piecemeal instead of needing to do a massive update for the entire repo at once